src/shared/components/footer/footer.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="footer">
+  <div class="footer-info">
+    <svg-icon [src]="'shoperz-logo'"></svg-icon>
+    <div>
+      <h5 class="footer-info-title">headquarters</h5>
+      <p>Presei Square No45, Bucharest - 099455</p>
+    </div>
+    <div>
+      <h5 class="footer-info-title">email</h5>
+      <p>contact@shoperz.com</p>
+    </div>
+    <div>
+      <h5 class="footer-info-title">telephone</h5>
+      <p>(+40) 987 123 654</p>
+    </div>
+    <div class="footer-info-icons">
+      <a class="footer-info-icon"
+         href="https://www.facebook.com/"
+         target="_blank">
+        <svg-icon [src]="'facebook-logo'"></svg-icon>
+      </a>
+      <a class="footer-info-icon"
+         href="https://www.instagram.com/"
```

Comment:
add all links to the environment.ts file,
add the `redirectUrls: {}` object there and put all links to it.